### DISCUSSION TITLE
请问你是怎么host页面的？不是需要 uername/username.github.io吗 谢谢

### DISCUSSION THREAD

**Discussion by itslijohnny** [adoyle-h/Today-I-Learned#9] [General]
请问你是怎么host页面的？不是需要 uername/username.github.io吗 谢谢

**Comment by adoyle-h** [comment]
> 请问你是怎么host页面的？不是需要 uername/username.github.io吗 谢谢

在 DNS 供应商那创建 CNAME，til.adoyle.me 指向 adoyle-h.github.io。

详见 https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site#configuring-a-subdomain

**Comment by itslijohnny** [comment]
谢谢！不知为什么，我没设置什么， /til的就能访问了

**Reply by adoyle-h** [reply]
你用 username.github.io 的域名访问，默认是这样的。 username.github.io/repo

**Comment by itslijohnny** [comment]
你的这个repo是Today-I-Learned 应该是 til.adoyle.me 指向 adoyle-h.github.io/Today-I-Learned 对吧

**Reply by adoyle-h** [reply]
并不是，就是我说的 CNAME。
然后官方文档写的很清楚啊。

In the "Code and automation" section of the sidebar, click  Pages.
Under "Custom domain", type your custom domain, then click Save.

**Comment by itslijohnny** [comment]
目前我有个domain（lijohnny.com） 已经指向iamjohnnyli/iamjohnnyli.github.io，然后iamjohnnyli/Today-I-Learned这个repo就自动发布到 http://lijohnny.com/Today-I-Learned/了.
